__fixtures__/inputs/ca-cvc-vaccination-certicate.json

```diff
@@ -0,0 +1,36 @@
+{
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3c-ccg.github.io/vaccination-vocab/context/v1/CA/unstable.json",
```

Comment:
we should use a redirect here, something like https://w3id.org/vaccination/CA/unstable/v1 or similar.

Reply:
@OR13 Hopefully my comments on the other 2 issues you've highlighted explains the context a bit more here. We will work with @tplooker on the redirects set up.